src/debug_commands.c

```diff
@@ -1433,6 +1562,12 @@ DebugCommandType commands[] = {{"DUMP_INVIDX", DumpInvertedIndex}, // Print all
                                {"DUMP_HNSW", dumpHNSWData},
                                {"SET_MONITOR_EXPIRATION", setMonitorExpiration},
                                {"WORKERS", WorkerThreadsSwitch},
+                               {"SET_MAX_SCANNED_DOCS", setMaxScannedDocs},
```

Comment:
I would suggest putting all of this as subcommands under one debug command `BG_SCAN_CONTROLLER` or something like that (see how we do it in `WorkerThreadsSwitch`)